book-crawler/scripts/backup_db.py

```
@@ -0,0 +1,222 @@
+        cutoff_date = datetime.now() - timedelta(days=days)
+        
+        for backup_file in self.backup_dir.glob("backup_*.sql.gz"):
+            # 파일명에서 날짜 추출
+            try:
+                date_str = backup_file.stem.split('_')[3]  # YYYYMMDD 부분
+                file_date = datetime.strptime(date_str[:8], "%Y%m%d")
+                
+                if file_date < cutoff_date:
+                    os.remove(backup_file)
+                    logger.info(f"오래된 백업 삭제: {backup_file.name}")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)